Makefile

```diff
@@ -201,7 +201,7 @@ ENVTEST_VERSION ?= release-0.19
 GOLANGCI_LINT_VERSION ?= v1.54.2
 MOCKERY_VERSION ?= v2.42.2
 NILAWAY_VERSION ?= latest
-GINKGO_VERSION ?= latest
+GINKGO_VERSION ?= v2.21.0
```

Comment:
Probably should set up Renovate for this stuff.

Reply:
Yea this particular case could be set to pin the same version in gomod that is why it needed to be set